src/pages/Strategies/Strategies.container.js

```diff
@@ -6,12 +6,14 @@ import WSActions from '../../redux/actions/ws'
 
 import StrategiesPage from './Strategies'
 import { getAuthToken } from '../../redux/selectors/ws'
+import { getMarketsForExecution } from '../../redux/selectors/meta'
 
 const mapStateToProps = (state) => ({
   authToken: getAuthToken(state),
   firstLogin: getFirstLogin(state),
   isGuideActive: getGuideStatusForPage(state, STRATEGY_PAGE),
   strategyContent: state.ui.content,
+  markets: getMarketsForExecution(state)
```

Comment:
```suggestion
  markets: getMarketsForExecution(state),
```

Trailing comma should be added to fix linting issue:
```
Failed to compile.

./src/pages/Strategies/Strategies.container.js
src/pages/Strategies/Strategies.container.js
  Line 16:41:  Missing trailing comma  comma-dangle

Search for the keywords to learn more about each error.
```